docs/contributing.md

```diff
@@ -76,8 +76,15 @@ scripts which maximally automate the process:
   usage.
   
 - **Testing**
+  For the test suite to run you need a current version of Chrome installed.
   Use `node ./run test` run both unit and web-based visual test.
   
+  - *Note for Windows users*:
+    there is a [known issue with wasm-pack](https://github.com/rustwasm/wasm-pack/issues/611) using the
+    wrong version of the chrome driver. There is
+    [a workaround](https://github.com/rustwasm/wasm-pack/issues/611#issuecomment-522093207) described in 
+    the issue. The easiest fix is to run `npm install -g chromedriver`.
```

Comment:
I would not call a global installation of a dependency to be a fix. We should not encourage anyone to install deps globally in their system (for some people it could not even be possible btw if they do not have admin rights).

Reply:
I can change it to recommend the more generic "install a compatible chrome driver and ensure it's in your `PATH`" advice instead.

Reply:
TBH I would strongly prefer the previous advice. It gives me at least actionable item. I can copy-paste command and proceed. With the current wording, I can't.

Can we keep both of them ? (if needed just add warning that npm command affects user's global state)

> (for some people it could not even be possible btw if they do not have admin rights).
Does isntalling npm package can ever require admin rights? The storage is user-specific, at least on Windows.

Reply:
I think having either both of them or none and just referring to the other issue, might be a good solution. The `npm` one is definitly the quickest way to resolve this.